actions/send.email.yaml

```diff
@@ -0,0 +1,57 @@
+---
+description: Send Email
+enabled: true
+entry_point: src/send_email.py
+name: send.email
+parameters:
+    send_as_html:
+        description: "If True Send Email as HTML, else as plain text"
+        required: true
+        type: boolean
+        default: true
+    admin_override:
+        description: "Override sending emails to instead send to one test email address"
+        required: true
+        type: boolean
+        default: false
+    admin_override_email:
+        description: "If admin_override enabled, will send all emails to this instead of to specified sender"
+        required: true
+        type: array
+    smtp_account:
+        description: "SMTP Account to use. Must be configured in email.yaml"
+        required: true
+        type: string
+    email_to:
+        description: "Recipient Email Addresses"
+        required: true
+        type: array
+    email_from:
+        description: "Sender Email Address"

```

Comment:
I'm not keen on this parameter, it completely opens us to spoofing. I'd rather hardcode something like `cloudsupport` or `stackstorm` as the sender